oslat-client

```diff
@@ -78,6 +78,20 @@ while true; do
             cpu_main_thread=$1
             shift
             ;;
+        --bucket-size)
+           shift;
+           bucket_size=" --bucket-size $1"
+           shift;
+           ;;
+        --bucket-width)
+           shift;
+           bucket_width=" --bucket-width $1"
+           shift;
+           ;;
+        --zero-omit)
+           shift;
+           zero_omit=" --zero-omit"
+           ;;
```

Comment:
Do you mean to surface this for user control?  If so it either needs to be a `key=value` like `zero-omit=yes|no` or if you really, really just want a `key` you can do something like we have done in places like fio where we use `ON|OFF` and then remove the value with a param regex at rickshaw execution time:

https://github.com/perftool-incubator/bench-fio/blob/master/multiplex.json#L120-L124
https://github.com/perftool-incubator/bench-fio/blob/master/rickshaw.json#L22-L23